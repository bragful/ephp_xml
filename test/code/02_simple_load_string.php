<?php

$data = file_get_contents("test/films.xml");
var_dump(simplexml_load_string($data));

