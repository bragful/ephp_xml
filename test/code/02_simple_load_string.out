object(SimpleXMLElement)#1 (1) {
  ["film"]=>
  array(3) {
    [0]=>
    object(SimpleXMLElement)#2 (1) {
      ["@attributes"]=>
      array(1) {
        ["title"]=>
        string(18) "Back to the Future"
      }
    }
    [1]=>
    object(SimpleXMLElement)#3 (1) {
      ["@attributes"]=>
      array(1) {
        ["title"]=>
        string(24) "Raiders for the Lost Arc"
      }
    }
    [2]=>
    object(SimpleXMLElement)#4 (1) {
      ["@attributes"]=>
      array(1) {
        ["title"]=>
        string(23) "Ferris Buller's Day Off"
      }
    }
  }
}
